dts/bindings/spi/arm,pl022.yaml

```diff
@@ -0,0 +1,18 @@
+# Copyright (c) 2022 TOKITA Hiroshi <tokita.hiroshi@fujitsu.com>
+# SPDX-License-Identifier: Apache-2.0
+
+description: ARM PL022 SPI
+
+compatible: "arm,pl022"
+
+include: [spi-controller.yaml, pinctrl-device.yaml]
+
+properties:
+  reg:
+    required: true
+
+  clocks:
+    required: true
+
+  interrupts:
```

Comment:
`interrupt` is `false` by default so not needed.

Reply:
Removed it.